includes/admin/helper/views/html-oauth-start.php

```diff
@@ -10,7 +10,6 @@
 ?>
 
 <div class="wrap woocommerce wc_addons_wrap wc-helper">
-	<?php require WC_Helper::get_view_filename( 'html-section-nav.php' ); ?>
```

Comment:
Should we remove this file if we are not going to be using the tabs anymore?

Reply:
Removed it now.